services-custom/dynamodb-enhanced/src/main/java/software/amazon/awssdk/enhanced/dynamodb/DynamoDbAsyncTable.java

```diff
@@ -345,56 +345,61 @@
      * further details and constraints.
      * <p>
      * Example:
+     * <p>
+     * 1) Subscribing to {@link Page}s
+     * <pre>
+     * {@code
+     *
+     * QueryConditional queryConditional = QueryConditional.keyEqualTo(Key.builder().partitionValue("id-value").build());
+     * PagePublisher<Page<MyItem>> publisher = mappedTable.query(QueryEnhancedRequest.builder()
+     *                                                                              .queryConditional(queryConditional)
+     *                                                                              .build());
+     * publisher.subscribe(page -> page.items().forEach(item -> System.out.println(item)));
+     * }
+     * <p>
+     * 2) Subscribing to items across all pages
      * <pre>
      * {@code
      *
      * QueryConditional queryConditional = QueryConditional.keyEqualTo(Key.builder().partitionValue("id-value").build());
-     * SdkPublisher<Page<MyItem>> publisher = mappedTable.query(QueryEnhancedRequest.builder()
+     * PagePublisher<MyItem> publisher = mappedTable.query(QueryEnhancedRequest.builder()
```

Comment:
I think there's a .items() missing?

Reply:
Good catch. I wish there was a better way to check the compiler errors in the code snippets from javadocs. I'll go over all code snippets.